### PR TITLE
request new unit to be added:  unit:TONNE-PER-HA-PER-YEAR

### PR DESCRIPTION
**Pull request by Lachlan-Charles**

### CODE REVIEW

**Review by steveraysteveray — CHANGES_REQUESTED** [review]
Please take a look at our naming rules [here](https://github.com/qudt/qudt-public-repo/wiki/Unit-Vocabulary-Submission-Guidelines#qname-naming-rules), and the discussion that pertains to your example [here](https://github.com/qudt/qudt-public-repo/discussions/129). The correct URI for your new entry would be unit:TONNE-PER-HA-YEAR. 
Secondly, you have not adjusted the DimensionVector (should have T-1, not T0), nor the QuantityKind (should be quantitykind:MassPerAreaTime).

**Review by steveraysteveray — COMMENTED** [review]
Please also note that I overlooked that unit:YEAR does not exist. The URI for a year is unit:YR.

**Comment by jhodgesatmb** [comment]
Agreed

Jack Hodges, Ph.D.
Arbor Studios

**Comment by Lachlan-Charles** [comment]
Hi Steve,

Thanks for the email - sorry about the errors.  I have amended them and committed the changes - although it does not appear to be showing up in the repository (not sure if this is normal - I am new to Github).  If I did something incorrect please let me know and I will try again.

Thanks,

Lachlan

________________________________

File: vocab/unit/VOCAB_QUDT-UNITS-ALL-v2.1.ttl [hunk 1]
```diff
@@ -22097,6 +22097,16 @@ unit:TONNE-PER-HA
   rdfs:label "tonne per hectare"@en ;
   owl:sameAs unit:TON_Metric-PER-HA ;
 .
+unit:TONNE-PER-HA-YR
+  a qudt:DerivedUnit ;
+  a qudt:Unit ;
+  qudt:conversionMultiplier 0.1 ;
```
Comment (steveraysteveray) [inline vocab/unit/VOCAB_QUDT-UNITS-ALL-v2.1.ttl:22103]: Things are much improved! I have now taken a look at the conversionMultiplier. Please check my math, but I think the correct value should be approximately 3.17 x 10^-9, or 0.00000000317
This is the number needed to convert to KiloGM-PER-M2-SEC

**Review by steveraysteveray — CHANGES_REQUESTED** [review]

File: vocab/unit/VOCAB_QUDT-UNITS-ALL-v2.1.ttl (hunk 1 above)
Comment (Lachlan-Charles) [reply vocab/unit/VOCAB_QUDT-UNITS-ALL-v2.1.ttl:22103]: Hi Steve - I have amended the conversionMultiplier to read 0.00000000317. (your math is correct)

Thanks again

**Review by steveraysteveray — APPROVED** [review]

**Pull request merged**